examples/kind-quickstart/02-Client.ipynb

```diff
@@ -0,0 +1,606 @@
+      "    store.push(\"driver_stats_push_source\", event_df, to=PushMode.ONLINE_AND_OFFLINE)\n",
+      "  File \"/Users/dmartino/miniconda3/envs/feast3.11/lib/python3.11/site-packages/feast/feature_store.py\", line 1423, in push\n",
+      "    self.write_to_online_store(\n",
+      "  File \"/Users/dmartino/miniconda3/envs/feast3.11/lib/python3.11/site-packages/feast/feature_store.py\", line 1449, in write_to_online_store\n",
+      "    feature_view: FeatureView = self.get_stream_feature_view(\n",
+      "                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
+      "  File \"/Users/dmartino/miniconda3/envs/feast3.11/lib/python3.11/site-packages/feast/feature_store.py\", line 504, in get_stream_feature_view\n",
+      "    return self._get_stream_feature_view(\n",
+      "           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
+      "  File \"/Users/dmartino/miniconda3/envs/feast3.11/lib/python3.11/site-packages/feast/feature_store.py\", line 514, in _get_stream_feature_view\n",
+      "    stream_feature_view = self._registry.get_stream_feature_view(\n",
+      "                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
+      "  File \"/Users/dmartino/miniconda3/envs/feast3.11/lib/python3.11/site-packages/feast/infra/registry/remote.py\", line 209, in get_stream_feature_view\n",
+      "    response = self.stub.GetStreamFeatureView(request)\n",
+      "               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
+      "  File \"/Users/dmartino/miniconda3/envs/feast3.11/lib/python3.11/site-packages/grpc/_channel.py\", line 1181, in __call__\n",
+      "    return _end_unary_response_blocking(state, call, False, None)\n",
+      "           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
+      "  File \"/Users/dmartino/miniconda3/envs/feast3.11/lib/python3.11/site-packages/grpc/_channel.py\", line 1006, in _end_unary_response_blocking\n",
+      "    raise _InactiveRpcError(state)  # pytype: disable=not-instantiable\n",
+      "    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
+      "grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:\n",
+      "\tstatus = StatusCode.UNKNOWN\n",
+      "\tdetails = \"Exception calling application: Feature view driver_hourly_stats_fresh does not exist in project sample\"\n",
+      "\tdebug_error_string = \"UNKNOWN:Error received from peer  {grpc_message:\"Exception calling application: Feature view driver_hourly_stats_fresh does not exist in project sample\", grpc_status:2, created_time:\"2024-09-17T13:19:54.127834+02:00\"}\"\n",
+      ">\n"
+     ]
+    }
+   ],
+   "source": [
+    "!cd client && python test_workflow.py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note If you see the following error, it is likely due to the [issue #4392](https://github.com/feast-dev/feast/issues/4392):\n",
```

Comment:
Looks like you're showing this error on purpose then, eh?

Reply:
Yep, the issue has been fixed after this PR, I should have updated it.